src/azure-cli/azure/cli/command_modules/monitor/tests/latest/test_monitor_metric_alert_scenarios.py

```diff
@@ -78,6 +78,30 @@ def test_metric_alert_v2_scenario(self, resource_group, storage_account):
             self.check('criteria.allOf[0].dimensions[0].values[0]', '*')
         ])
 
+    @ResourceGroupPreparer(name_prefix='cli_test_metric_alert_special_char')
+    def test_metric_alert_special_char_scenario(self, resource_group):
+        self.kwargs.update({
+            'alert': 'alert1',
+            'rg': resource_group
```

Comment:
Is it enough to just use {rg} directly instead of adding it to update() explicitly?